client-side/package.json

```diff
@@ -81,17 +90,23 @@
     "@storybook/react-webpack5": "^8.1.11",
     "@storybook/test": "^8.1.11",
     "babel-jest": "^29.7.0",
+    "css-loader": "^7.1.2",
     "eslint-config-react-app": "^7.0.1",
     "eslint-plugin-react": "^7.27.1",
     "eslint-plugin-storybook": "^0.8.0",
     "globals": "^15.8.0",
-    "identity-obj-proxy": "^3.0.0",
-    "jest-environment-jsdom": "^29.7.0",
-    "jest-transform-stub": "^2.0.0",
```

Comment:
why did you removed:
"identity-obj-proxy": "^3.0.0",
"jest-transform-stub": "^2.0.0",

we need them for running the tests as well